### DISCUSSION TITLE
My solar production icon still remains

### DISCUSSION THREAD

**Discussion by R0DRIG0MEL0** [slipx06/sunsynk-power-flow-card#262] [General]
I'm having this problem in the calculations, my solar production icon still remains 
![Screenshot_20240219_173238_Home Assistant](https://github.com/slipx06/sunsynk-power-flow-card/assets/60002358/4452a69d-a121-4e51-b042-3ca0c0daace6)

**Comment by R0DRIG0MEL0** [comment]
Even though I have consumption on my dealership network, my home icon still has full production from the solar panel, see that the autarkia also reports 100%

**Comment by slipx06** [comment]
Hi. Sorry I'm not sure what you are asking? Which solar production icon? The home icon?

**Reply by R0DRIG0MEL0** [reply]
yes, the house icon, at the moment when I have little consumption from the grid, the icon still continues to be reported as solar production

**Comment by R0DRIG0MEL0** [comment]
![Captura de tela 2024-02-20 125503](https://github.com/slipx06/sunsynk-power-flow-card/assets/60002358/478592d5-6249-4ee1-8128-ce9685a0b575)
see this other case, the house icon is complete in yellow, informing solar energy, where it should be partial. It is believed to be due to the use of AUX, would it be possible to add the consumption of the house and aux in the calculation?

**Reply by slipx06** [reply]
The calculation is fairly simple and does not consider AUX power at the moment. I'll take another look

**Comment by slipx06** [comment]
@R0DRIG0MEL0 please test https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.16.1

**Comment by R0DRIG0MEL0** [comment]
It worked perfectly, you're an amazing guy, congratulations on your work, it's definitely the best homeassistants card. This is due to your work and attention, congratulations and thank you very much once again!

**Comment by R0DRIG0MEL0** [comment]
I think another important tip is when you are on the Mode Lite or Compact, the AUX is added to the essential power, this way the medication will be complete

**Reply by slipx06** [reply]
A quick fix is to create a template sensor that adds L1+L2+L3+ AUX and use that for essential_power on the lite/compact card. The original Sunsynk measurement for essential_power excludes the Auxiliary power

**Reply by R0DRIG0MEL0** [reply]
ok

**Comment by R0DRIG0MEL0** [comment]
I think I have another problem, the calculation and consumption of my house is based on the size of my panels, and not on the output power of my inverter
![Captura de tela 2024-02-23 140019](https://github.com/slipx06/sunsynk-power-flow-card/assets/60002358/cfb3824e-7a3c-44a7-a574-f37e8029368c)

**Reply by R0DRIG0MEL0** [reply]
I only refer to the icon of the house, only

**Reply by slipx06** [reply]
The source could be either battery or solar so we need to look at the input. I don't account for inverter losses so its not a 100% accurate but close enough.
